apisix/plugins/ocsp-stapling.lua

```diff
@@ -166,6 +179,7 @@ local function set_cert_and_key(sni, value)
     end
 
     local ok, err = set_ocsp_resp(fin_pem_cert,
+                                  value.ocsp_stapling.ssl_stapling_responder,
```

Comment:
this value is being passed around through many functions, I would recommend checking if this value is nil. If it is nil, use `get_ocsp_responder`.

Reply:
this value will always be the params of function `get_ocsp_responder` on line 49, the function will do a check.

Reply:
yes but instead of passing this values through mulitple functions, I think a simple check before hand can avoid this and make code simpler.

Reply:
ok